flink-runtime-web/web-dashboard/src/app/share/common/dagre/node.component.ts

```diff
@@ -31,8 +31,18 @@ export class NodeComponent {
   operatorStrategy: string | null;
   parallelism: number | null;
   lowWatermark: number | null | undefined;
+  backPressuredPercentage: number | undefined = NaN;
+  busyPercentage: number | undefined = NaN;
+  backgroundColor: string | undefined;
+  borderColor: string | undefined;
   height = 0;
   id: string;
+  lightRed = '#ee6464';
+  lightBlue = '#5db1ff';
+  lightBlack = '#888888';
+  red = '#ee2222';
+  blue = '#1890ff';
+  black = '#000000';
```

Comment:
I think it's better to name these colors semantically, e.g. back-pressuring instead of red, back-pressured instead of black.